OpenSim/Simulation/Manager/Manager.cpp

```diff
@@ -50,7 +50,7 @@ std::string Manager::_displayName = "Simulator";
 //=============================================================================
 Manager::Manager(Model& model) : Manager(model, true)
 {
-    _integ = new SimTK::RungeKuttaMersonIntegrator(_model->getMultibodySystem());
+    _integ = make_unique<SimTK::RungeKuttaMersonIntegrator>(_model->getMultibodySystem());
```

Comment:
The function template `std::make_unique` is C++14, and we are still only on C++11. Could you use `_integ.reset(new ...)` instead?

Reply:
Absolutely. Travis also strongly agrees with you already...